db-tests/src/test/java/TestTriggers.java

```diff
@@ -2,6 +2,7 @@
 import com.zaxxer.hikari.HikariConfig;
 import com.zaxxer.hikari.HikariDataSource;
 import org.junit.Assume;
```

Comment:
My only comment for this PR 😆:  You could flip the order of these two imports and then you'd have a lovely alphabetically ordered list.

Reply:
Removed Assume in favor of Assumptions